ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -59,6 +59,7 @@ tests:
     workflow: openshift-e2e-gcp
 - as: e2e-gcp-ovn-upgrade
   commands: CLUSTER_NETWORK_TYPE=OVNKubernetes TEST_SUITE=all run-upgrade-tests
+  cron: '@yearly'
```

Comment:
This job needs a comment explaining why it's here (in each file). Also, the `CLUSTER_NETWORK_TYPE=OVNKubernetes` in the `commands` line is a no-op, right? Probably less confusing overall if you remove that then.

Reply:
@danwinship you can't do comments here, I tried, and the 'make' stuff pulled them out